packages/plugins/integration-zapier/src/lib/zapier.service.ts

```
@@ -57,72 +59,67 @@ export class ZapierService {
-						current.settingsName === 'client_secret' ? current.settingsValue : prev.client_secret,
-					refresh_token: current.settingsName === 'refresh_token' ? current.settingsValue : prev.refresh_token
-				};
-			},
-			{
-				client_id: '',
-				client_secret: '',
-				refresh_token: ''
+			const { client_id, client_secret, refresh_token } = settings.reduce(
+				(prev, current) => {
+					return {
+						...prev,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)